ui/components/component-library/badge-wrapper/README.mdx

```diff
@@ -0,0 +1,396 @@
+- top right `BadgeWrapperPosition.topRight`
+- bottom left `BadgeWrapperPosition.bottomLeft`
+- bottom right `BadgeWrapperPosition.bottomRight`
+
+If you require a custom position, you can use the `positionObj` prop see [Position Obj](/docs/components-componentlibrary-badgewrapper--position-obj#position-obj) for more details.
+
+<Canvas>
+  <Story id="components-componentlibrary-badgewrapper--position" />
+</Canvas>
+
+```jsx
+import { BorderColor Size } from '../../../helpers/constants/design-system';
+
+import {
+  AvatarAccount,
+  AvatarNetwork,
+  BadgeWrapper,
+  BadgeWrapperPosition
+} from '../../component-library';
+
+<BadgeWrapper
+  position={BadgeWrapperPosition.topLeft}
+  badge={
+    <AvatarNetwork
+      size={Size.XS}
+      name="Arbitrum One"
+      src="./images/avax-token.png"
+      borderColor={BorderColor.borderMuted}
+    />
+  }
+>
+  <AvatarAccount
+    address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+    size={Size.MD}
+  />
+</BadgeWrapper>
+<BadgeWrapper
+  badge={
+    <AvatarNetwork
+      size={Size.XS}
+      name="Arbitrum One"
+      src="./images/avax-token.png"
+      borderColor={BorderColor.borderMuted}
+    />
+  }
+>
+  <AvatarAccount
+    address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+    size={Size.MD}
+  />
+</BadgeWrapper>
+<BadgeWrapper
+  position={BadgeWrapperPosition.bottomLeft}
+  badge={
+    <AvatarNetwork
+      size={Size.XS}
+      name="Arbitrum One"
+      src="./images/avax-token.png"
+      borderColor={BorderColor.borderMuted}
+    />
+  }
+>
+  <AvatarAccount
+    address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+    size={Size.MD}
+  />
+</BadgeWrapper>
+<BadgeWrapper
+  position={BadgeWrapperPosition.bottomRight}
+  badge={
+    <AvatarNetwork
+      size={Size.XS}
+      name="Arbitrum One"
+      src="./images/avax-token.png"
+      borderColor={BorderColor.borderMuted}
+    />
+  }
+>
+  <AvatarAccount
+    address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+    size={Size.MD}
+  />
+</BadgeWrapper>
+```
+
+### Position Obj
+
+Use the `positionObj` prop to set a custom position for the badge. The `positionObj` prop takes an object with the following properties:
+
+- `top` - the top position of the badge
+- `right` - the right position of the badge
+- `bottom` - the bottom position of the badge
+- `left` - the left position of the badge
+
+<Canvas>
+  <Story id="components-componentlibrary-badgewrapper--position-obj" />
+</Canvas>
+
+```jsx
+import { BorderColor Size } from '../../../helpers/constants/design-system';
+
+import {
+  AvatarAccount,
+  AvatarNetwork,
+  BadgeWrapper,
+} from '../../component-library';
+
+
+<BadgeWrapper
+  badge={
+    <AvatarNetwork
+      size={Size.XS}
+      name="Arbitrum One"
+      src="./images/avax-token.png"
+      borderColor={BorderColor.borderMuted}
+    />
+  }
+  positionObj={{ top: 4, right: -8 }}
+>
+  <AvatarAccount
+    address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+    size={Size.SM}
```

Comment:
Can we make this default size to `size={Size.MD}` to keep it in sync with the other stories?

Reply:
Yes! Fixed here https://github.com/MetaMask/metamask-extension/pull/17851/commits/d01346d0cff09d59c6741d4c6b2e3e0bc7f36bd8